setup.cfg

```diff
@@ -30,7 +30,7 @@ install_requires =
     packaging
     pybids >= 0.10.2
     pyyaml
-    templateflow >= 0.5.2
+    templateflow ~= 0.6
```

Comment:
it seems we keep flip flopping on `~=` vs `>=`

Reply:
I would make sure we fulfill poldracklab/fmriprep#2054, and since TemplateFlow is not there there is a little more flexibility.

With the integration of the auto-update, it feels safe to say `~= 0.6`. For the TemplateFlow client there is no API change in the horizon that would affect fMRIPrep. If any is introduced, we should/can start its 1.x series (and therefore the `~=` selector would nicely work out).